### PR TITLE
Localize buttons

### PR DESCRIPTION
**Pull request by tannerj**
Adds localizations for buttons and navigation links in all view files.

### CODE REVIEW

**Comment by monfresh** [comment]
I think this looks great so far. Could you please do 3 more things:

- Remove the trailing whitespace in `config/locales/en.yml`
- Update the specs that check for the text you localized so that they verify the localization as opposed to the text itself. For example, in https://github.com/codeforamerica/ohana-api/blob/master/spec/features/admin/dashboard_spec.rb#L123, replace:
```ruby
expect(page).to have_link 'Services'
```
with
```ruby
expect(page).to have_link t('admin.navigation.services')
```

- When you're done, please squash your commits into one. Let me know if you need help doing that.

Thanks!

**Comment by tannerj** [comment]
I think I have addressed the three issues you raised. I also removed the calls to admin.navigation and replaced them with calls to admin.buttons in config/locales/en.yml because they were repeating the same text. I'm not sure if this is the way you want this done. Another option would be referencing one in the other.  Please let me know how you'd like that structured and I'll add that change in my next pull request. Thanks for being patient with me on this.

**Comment by monfresh** [comment]
Thanks so much for doing this! At first glance, this looks perfect.

File: app/views/admin/locations/_confirm_delete_location.html.haml [hunk 1]
```diff
@@ -19,4 +19,4 @@
         permanently.
     .modal-footer
       %button.btn.btn-default{ 'data-dismiss' => 'modal' } Close
```
Comment (md5) [inline app/views/admin/locations/_confirm_delete_location.html.haml]: Should "Close" be localized here?

Comment (monfresh) [inline app/views/admin/locations/_confirm_delete_location.html.haml]: Yes, good catch!

Comment (monfresh) [inline app/views/admin/locations/_confirm_delete_location.html.haml]: We should also move that to a shared partial as the same code is used on other delete views.

**Pull request merged**